sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -802,6 +804,35 @@ def extract_output(self, accumulator):
     return accumulator
 
 
+class ToSet(ptransform.PTransform):
+  """A global CombineFn that condenses a PCollection into a set."""
+
+  def __init__(self, label='ToSet'):  # pylint: disable=useless-super-delegation
```

Comment:
Is this pylint directive still required with the version of pylint Beam is using today?

Reply:
Copied from line 775 (now line 777). Do you want me to experiment? A change removing this seems like it should be a separate change.

Reply:
Removing this produces
************* Module apache_beam.transforms.combiners
apache_beam/transforms/combiners.py:810:2: W0235: Useless super delegation in method '__init__' (useless-super-delegation)

------------------------------------

Reply:
Thanks for checking.